src/main.cpp

```diff
@@ -32,124 +32,35 @@ void loop()
-    Serial.println(F("Radio Mode CW, please check"));
-    delay(dly);
-
-    radio.setMode(CAT_MODE_CWR); // set mode to CW-R
-    Serial.println(F("Radio Mode CW-R, please check"));
-    delay(dly);
-
-    radio.setMode(CAT_MODE_DIG); // set mode to DIG
-    Serial.println(F("Radio Mode DIG, please check"));
-    delay(dly);
-
-    radio.setMode(CAT_MODE_FM); // set mode to FM
-    Serial.println(F("Radio Mode FM, please check"));
-    delay(dly);
-
-    radio.setFreq(10150000);
-    radio.setMode(CAT_MODE_WBFM); // set mode to WBFM
-    Serial.println(F("Radio Mode WBFM (Broadcast band), please check"));
-    delay(dly);
-
-    radio.setFreq(1407000); // set VFO frequency to xx
-    radio.setMode(CAT_MODE_USB); // set mode to USB
-    Serial.println(F("Radio set to USB in 14.070Mhz, please check"));
-    delay(dly);
-
-    // perform a frequency sweep
-    Serial.println(F("Doing a sweep up in the 2m band, please check"));
-    radio.setMode(CAT_MODE_FM); // set mode to FM
-    for (long freq = 0; freq<100; freq++)
-    {
-        radio.setFreq(14500000 + freq * 500);
-        delay(50);
-    }
-    delay(dly);
-
-    radio.clar(true); // enable clarifier
-    Serial.println(F("Radio Clarifier ON, please check"));
-    delay(dly);
-
-    radio.clar(false); // disable clarifier
-    Serial.println(F("Radio Clarifier OFF, please check"));
-    delay(dly);
-
-    radio.split(true); // enable split operation
-    Serial.println(F("Radio Split ON, please check"));
-    delay(dly);
-
-    radio.split(false); // disable split operation
-    Serial.println(F("Radio Split OFF, please check"));
-    delay(dly);
-
-    radio.rptrOffset("+"); // set positive repeater offset
-    Serial.println(F("Radio Positive repeater offset, please check"));
-    delay(dly);
-
-    radio.rptrOffset("-"); // set negative repeater offset
-    Serial.println(F("Radio Negative repeater offset, please check"));
-    delay(dly);
-
-    radio.rptrOffsetFreq(600); // set an offset frequency in kHz
-    Serial.println(F("Radio repeater offset =  600 Khz, please check"));
-    delay(dly);
-
-    radio.rptrOffset("s"); // enable simplex mode
-    Serial.println(F("Radio Simplex mode, please check"));
-    delay(dly);
-
-    radio.squelch("TSQ"); // set the radio to CTCSS on RX & TX
-    Serial.println(F("Radio TSQ ON, please check"));
-    delay(dly);
-
-    radio.squelchFreq(1000, "C"); // set CTCSS tone 100.0Hz (TX + RX) - note the C = CTCSS tone
-    Serial.println(F("Radio TSQ CTSS, please check"));
-    delay(dly);
-
-    radio.squelch("DCS"); // set the radio to DCS on RX & TX
-    Serial.println(F("Radio DCS ON, please check"));
-    delay(dly);
-
-    radio.squelchFreq(371, "D"); // set DCS code 0371 (TX + RX) - note the D = DCS code
-    Serial.println(F("Radio DCS code 0371, please check"));
-    delay(dly);
-
-    radio.squelch("OFF"); // clear CTCSS & DCS squelch
-    Serial.println(F("Radio TSQ OFF, please check"));
-    delay(dly);
```

Comment:
Most of these looked like they were working first time (apart form the setMode items but that's fixed and tested, I assume the VE3BUX mode enumeration is good)